src/luna/common/utils.py

```diff
@@ -253,9 +253,8 @@ def validate_params(given_params: dict, params_list: List[tuple]):
     d_params = {}
     for param, dtype in params_list:
         if given_params.get(param, None) is None:
-            raise RuntimeError(
```

Comment:
Raising an error here is incorrect behavior if the parameter is optional.

Reply:
(optional as declared by the click decorator)

Reply:
Gotcha, this makes sense